### ISSUE TITLE
Can't attack Tiamat while grounded

### ISSUE THREAD

**Issue by password100** [LegionDark/Issues#903]
**Date & Time**: 
11:09 AM PST

**FFXI Client Version (use `/ver`)**:
 30160329_1

**Character Name**: 
Monday

**Nation**: 
Sandoria

**Job(level)/Sub Job(level)**: 
99War/49Sam

**NPC or Monster or item Name**: 
Tiamat

**Zone name & Coordinates (use `@where`)**: 
Attowha Chasm - X: -515.011, Y: -12.732, Z: -78.101

**ffxiah.com link (for items issues only)**: 

**Multi-boxing? (multiple clients on same connection)**: 
Yes

**Steps To Reproduce / any other info**: 
Ran in to attack him while he was grounded but seemed as if he had his flying mode status because I couldn't attack him at all and couldn't even get a trick attack to hit on my THF alt.

(in the SS he was grounded the whole time also) not sure why this has happened because he was zerged down a couple days prior. Sorry I don't have an actual SS of him actually grounded but you'll just have to take my word for it and the people we were supposedly stalking.
![monday_2016 04 24_140010](https://cloud.githubusercontent.com/assets/17345448/14770434/6a951636-0a26-11e6-8dc9-a89d904e4d07.png)

On another note (please feel free to ignore this if this isn't the place for this as I will just contact a GM in game if so) -  This may be irrelevant to the bug but I also need clarification on what is considered "rude" or "stalking" because we were definitely at the site of Tiamat first and were forced to leave as we were regrouping as different jobs and coming back after the wipe because what we were doing was considered "Rude" behavior because we were "Stalking" the other group who had most obviously followed us to Attohwa Chasm as they saw me use the Homepoint to Boneyard Gully although I have no proof of this you can probably assume so from looking at the log. 

Also I don't know why we advised to leave when we wanted to leave a mule there to watch and see if the other party would wipe or at least get ToD for the next day. Nevertheless if cheering on a bugged NM while we're wiping isn't considered rude, then obviously we're not on the same page in terms of the definition of "rude". 

I just want say I have nothing against anyone but I do believe FFXI is a competitive game and I played during the 2005-2008 Era of camping HNM and still play the game the same way as when I've first started so I also need clarification on "stalking" because I don't know how else to get ToD. I'm a bit frustrated at the moment and felt I needed to add this.

Sorry for the rant and thank you for your time.

**Comment by R3P0FFXI** [comment]
@password100 confirmed this as a bug @TeoTwawki thoughts?

**Comment by Samikana** [comment]
@password100 No one forced you to leave.  Thehulk sent me a tell asking if I thought he could stay or not to spectate their battle.  I told him that if your group didn't plan to fight Tiamat, I wouldn't suggest staying, as it could cause issues later down the line.  Not at any point did I tell you or your group they were not welcome at Tiamat.

I wasn't taking anyone's word for if they were or weren't there first, and my announcement was because I've had multiple complaints about your group following other players to tantalize them and I've watched while hidden and seen exactly what was described to me in tells.  

Thehulk also claimed that the other group was following them.  I have only seen one group follow the other during the occasions that I get a complaint about it, and I told him that he is always welcome to send me a tell if he feels he's genuinely being followed with the intention of harassment.

This is a competitive game, and there is every reason to believe that some incidents are coincidental due to the fact that both of your groups may need the same gear.  However, two GMs now have speculated exactly what was being reported to us, which is why I took to making the announcement.  These reports are especially looked into (from both sides, not just one) due to the pre-existing animosity and resentment between the people in the groups.  

You're also welcome to send me a tell in-game if you don't understand or want to discuss this.  However, I am not changing my stance.  If I see any more harassment in-game, the penalty will be a 3 days in jail.  This goes to both your group and the other group, and if it happens to you, send me a tell immediately.

- Halsey
